README.md

```diff
@@ -177,3 +177,14 @@ Bug reports and pull requests are welcome on GitHub at https://github.com/AirHel
 ## License
 
 The gem is available as open source under the terms of the [MIT License](http://opensource.org/licenses/MIT).
+
+## Development mode 
+
+When developing an application with Eventboss one might want to test if processing received messages works as expected.
+
+In the development mode:
+* [ ] SNS topics required by the application are created on the fly
+* [ ] SQS queues are created on the fly
+* [ ] subscriptions are created for topics and queues
+* [ ] development mode must be explicitely enabled via env variable  
+* [ ] it works™
```

Comment:
That's not yet implemented, right?
That might be confusing, what about that we've wait with that when implemented?